src/main/java/com/digitalasset/testing/ledger/DamlScriptRunner.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2019 Digital Asset (Switzerland) GmbH and/or its affiliates
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.digitalasset.testing.ledger;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class DamlScriptRunner {
+  private final Logger logger = LoggerFactory.getLogger(getClass().getCanonicalName());
+
+  private final ProcessBuilder processBuilder;
+
+  private Process script;
+
+  private DamlScriptRunner(ProcessBuilder processBuilder) {
+    this.processBuilder = processBuilder;
+  }
+
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  public void run() throws IOException, InterruptedException {
+    logger.debug("Executing: {}", String.join(" ", processBuilder.command()));
+    script = processBuilder.start();
+    if (script.waitFor() != 0) {
+      throw new IllegalStateException("Unexpected termination of DAML script.");
+    }
+    logger.info("DAML Script has run successfully.");
+  }
+
+  public void kill() {
+    try {
+      if (script.isAlive()) {
+        script.destroyForcibly().waitFor();
+      }
+    } catch (InterruptedException e) {
+      logger.error("Could not stop DAML script.", e);
+    }
+  }
+
+  public static class Builder {
+
+    private String darPath;
+    private String scriptName;
+    private String sandboxPort;
+    private boolean useWallclockTime = false;
```

Comment:
```suggestion
    private boolean useWallClockTime = false;
```
and subsequent changes.

Reply:
okay